### PR TITLE
Improve release process 0.7.2

### PR DESCRIPTION
**Pull request by simon-mo**
- Add comments about stress test
- Remove sgd test because it uses deprecated sgd library
- Remove impala python2 test because openai gym removed python2 support
- Add a script get PR contributors' github login

### CODE REVIEW

File: dev/RELEASE_PROCESS.rst [hunk 1]
```diff
@@ -37,6 +37,8 @@ This document describes the process for creating new releases.
 
    and make sure they pass. If they pass, it will be obvious that they passed.
    This will use the autoscaler to start a bunch of machines and run some tests.
+   Stress test will use significant aws resource. If you don't have enough quota,
```
Comment (richardliaw) [inline dev/RELEASE_PROCESS.rst]:
```suggestion
   **Caution!**: By default, the stress tests will require expensive GPU instances.
```

File: dev/RELEASE_PROCESS.rst [hunk 2]
```diff
@@ -37,6 +37,8 @@ This document describes the process for creating new releases.
...
+   please contact Robert for account.
```
Comment (richardliaw) [inline dev/RELEASE_PROCESS.rst]:
```suggestion
```

File: dev/get_contributors.py [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    # Use Github API to fetch the
+    g = Github(access_token)
+    ray_repo = g.get_repo("ray-project/ray")
+    logins = set()
+    for num in tqdm(pr_numbers):
+        try:
+            logins.add(ray_repo.get_pull(num).user.login)
+        except Exception as e:
+            print(e)
+            pass
```
Comment (richardliaw) [inline dev/get_contributors.py]:
```suggestion
```

File: dev/RELEASE_PROCESS.rst [hunk 4]
```diff
@@ -127,7 +123,20 @@ This document describes the process for creating new releases.
       git pull origin master --tags
       git log $(git describe --tags --abbrev=0)..HEAD --pretty=format:"%s" | sort
 
-11. **Update version numbers throughout codebase:** Suppose we just released
+
+    At the end of the release note, you can add a list of contributors that help
+    creating this release. Use the ``dev/get_contributors.py`` to generate this
+    list. You will need to create a github token for this task. Example usage:
```
Comment (richardliaw) [inline dev/RELEASE_PROCESS.rst]:
```suggestion
    list. You will need to create a Github token for this task. Example usage:
```

**Review by richardliaw — APPROVED** [review]
Looks fine to me.

**Comment by simon-mo** [comment]
@robertnishihara @richardliaw can this be merged?

**Pull request merged**